src/Form/Form.js

```diff
@@ -87,65 +53,103 @@ class Form {
                 if (!entry.isIntersecting) this.removeTooltip()
             }
         })
+        this.categorizeInputs()
+    }
+
+    submitHandler () {
+        if (this.handlerExecuted) return
 
-        this.removeTooltip = (e) => {
-            const tooltip = this.device.getActiveTooltip()
-            if (
-                this.isAutofilling ||
-                !tooltip ||
-                (e && e.target === tooltip.host)
-            ) {
-                return
+        const credentials = this.getValues()
+        if (credentials.password) {
+            // ask to store credentials and/or fireproof
+            if (this.shouldPromptToStoreCredentials) {
+                // @ts-ignore
+                this.device.storeCredentials(credentials)
             }
-            this.device.removeTooltip()
-            this.intObs?.disconnect()
-            window.removeEventListener('pointerdown', this.removeTooltip, {capture: true})
```

Comment:
Removed event listener and any reference to `e` as we no longer have this pointer down handler.